L1Trigger/CSCTriggerPrimitives/src/CSCTriggerPrimitivesBuilder.h

```diff
@@ -81,14 +97,27 @@ class CSCTriggerPrimitivesBuilder
   /** SLHC: special switch for disabling ME42 */
   bool disableME42;
 
+  /** SLHC: special switch for the upgrade ME1/1 TMB */
+  bool runME11ILT_;
+
+  /** SLHC: special switch for the upgrade ME2/1 TMB */
+  bool runME21ILT_;
+
+  /** SLHC: special switch for the upgrade ME3/1 and ME4/1 TMB */
+  bool runME3141ILT_;
+
   int m_minBX, m_maxBX; // min and max BX to sort.
 
   /** Pointers to TMB processors for all possible chambers. */
   CSCMotherboard*
     tmb_[MAX_ENDCAPS][MAX_STATIONS][MAX_SECTORS][MAX_SUBSECTORS][MAX_CHAMBERS];
```

Comment:
I've changed everything to smart pointers.  I think all the code was actually fine because the destructors cleaned up, but now that's clearer and technically a little safer since the compiler will stop if a copy is made that would make two instances point to the same thing.
@davidlange6, I didn't change this pointer array though.  It's iterated through and the items deleted in the destructor.  I wasn't sure if the overhead of an array of unique_ptrs was too much (looking at the constants above I think the array has 864 elements), or if you would suggest another structure?